### PR TITLE
Update "Configure Azure Monitor for VMs on Azure Stack Hub" guide

### PR DESCRIPTION
**Pull request by wturner123**
Update PowerShell code block to work with both Windows and Linux VMs.

### CODE REVIEW

**Review by cblackuk — CHANGES_REQUESTED** [review]
I think we want(ed) to add the data sources to it.

```powershell
## Configure data sources
$EventLogNames = @()
$EventLogNames += "System"
$EventLogNames += "Application"
$EventLogNames += "Setup"
$Count = 0
foreach ($EventLogName in $EventLogNames) {
    $Count++
    $null = New-AzureRMOperationalInsightsWindowsEventDataSource `
        -ResourceGroupName $LogAnalyticsWorkspaceResourceGroupName `
        -WorkspaceName $LogAnalyticsWorkspaceName `
        -Name "$($EventLogName) Event Log" `
        -EventLogName $EventLogName `
        -CollectErrors `
        -CollectWarnings `
        -CollectInformation
}

```

**Review by cblackuk — APPROVED** [review]
LGTM

File: articles/azure/azs-how-setup-monitoring.md [hunk 1]
```diff
@@ -42,164 +42,387 @@ To complete the steps in this article, you must have appropriate access to a sub
...
+                      -ExtensionType $MonitoringAgentType `
+                      -TypeHandlerVersion $MonitoringAgentVersion `
+                      -SettingString $PublicSettings `
+                      -ProtectedSettingString $ProtectedSettings `
+                      -Verbose</code></pre>
 
 ***
 
-8. In public Azure, on the top left, click the hamburger icon and select **Monitor**.
+# [Portal](#tab/tabid-1)
```
Comment (suehighmoor) [inline articles/azure/azs-how-setup-monitoring.md:359]: Should there be a heading here to explain what these steps are for?

Comment (wturner123) [reply articles/azure/azs-how-setup-monitoring.md:364]: Agreed, added a header.

Comment (cblackuk) [reply articles/azure/azs-how-setup-monitoring.md:364]: LGTM

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**